package.json

```diff
@@ -25,7 +29,8 @@
   },
   "homepage": "https://github.com/fraserxu/electron-pdf",
   "devDependencies": {
-    "standard": "^6.0.6"
+    "jasmine": "^2.5.2",
```

Comment:
I have used Ava and I like it.  I will switch over to that if you prefer it as well.  It seems like it still stabilizing, but I think it will be fine for this project.